system-tests/projects/e2e/cypress/support/util.js

```diff
@@ -70,6 +71,8 @@ export const verify = (title, ctx, options) => {
       // code frames will show this as the 1st line
       if (isCyOrigin) {
         cy.get('.test-err-code-frame pre span').should('include.text', `('${title}',,function()`)
+      } else if (isPreprocessorWithTypescript) {
```

Comment:
has to do with using typescript in some of these on ts version 5 vs 5, which means the transpilation is slightly different. This is related to #29614